gr-audio/lib/osx/osx_sink.cc

```diff
@@ -859,10 +894,12 @@ int osx_sink::work(int noutput_items,
     }
 
 #if _OSX_AU_DEBUG_RENDER_
-    std::cerr << ((void*)(pthread_self())) << " : audio_osx_sink::work: "
+    std::ostringstream msg;
+    msg << "INFO " << ((void*)(pthread_self())) << " : audio_osx_sink::work: "
```

Comment:
same here: above we flush straight out of stderr, now we log via the INFO logger – do we want fastpath logging in all audio sinks?

Reply:
addressed in #3024